cfy_manager/components/restservice/db.py

```diff
@@ -35,17 +36,19 @@
 from ..service_components import DATABASE_SERVICE
 from ..service_names import (
     MANAGER,
-    POSTGRESQL_CLIENT,
     RABBITMQ,
     RESTSERVICE,
+    POSTGRESQL_CLIENT,
```

Comment:
Why not add POSTGRESQL_SERVER in alphabetical order the way POSTGRESQL_CLIENT was before?